### ISSUE TITLE
Import Ender 3 S1 Profile from Prusa Repo

### ISSUE THREAD

**Issue by bloodf** [supermerill/SuperSlicer#3164]
It will be awesome if the SuperSlicer could import the Creality Ender 3 S1 Profiles from Prusa repository.

**Comment by themaster567** [comment]
It's been three weeks and they still haven't done something that sure sound trivial. Is there any way this can be done manually?

**Comment by neophyl** [comment]
Yes, easily. A couple of ways. Open PRUSA slicer, select the printer, and the filament and print profile. Save a project file as 3mf as normal. Then open the project in super slicer. You can then save the 3 profiles manually just like saving any other profile.
